Workbooks/SapMonitor/MsSqlServer/MsSqlServer.workbook

```diff
@@ -1289,21 +1326,89 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let LastOne = MSSQL_AOFailovers_CL\r\n| where TimeGenerated {Days} and PROVIDER_INSTANCE_s == \"{Provider}\"\r\n| summarize TimeGenerated = max(TimeGenerated)\r\n| project TimeGenerated;\r\n\r\nMSSQL_AOFailovers_CL\r\n| join kind=inner (LastOne) on TimeGenerated\r\n| where PROVIDER_INSTANCE_s == '{Provider}'\r\n| project server_name_s, timestamp_t, Message\r\n\r\n",
+        "query": "let LastOne = MSSQL_AOWaitstats_CL\r\n| where TimeGenerated {Days} and PROVIDER_INSTANCE_s == \"{Provider}\"\r\n| summarize TimeGenerated = max(TimeGenerated)\r\n| project TimeGenerated;\r\n\r\nMSSQL_AOWaitstats_CL\r\n| join kind=inner (LastOne) on TimeGenerated\r\n| where PROVIDER_INSTANCE_s == \"{Provider}\"\r\n| project WaitType_s, Percentage_d, WaitCount_d, Wait_S_d, AvgWait_S_d,Resource_S_d, AvgRes_S_d, Signal_S_d, AvgSig_S_d, TimeGenerated\r\n| order by Percentage_d desc",
         "size": 3,
-        "title": "AlwaysOn Failovers",
+        "title": "AlwaysOn Waitstats",
         "noDataMessage": "No data.",
         "timeContext": {
           "durationMs": 86400000
         },
-        "queryType": 0
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
```

Comment:
Is this case insensitive? We usually follow something like this? Microsoft.OperationalInsights/workspaces

Reply:
Yes, I think it's insensitive since it worked :) Seems to be auto-generated